Week-2/index_2_2.css

```diff
@@ -0,0 +1,75 @@
+.box {
+    background: red;
+    width: 150px;
+    height: 200px;
+    transition: transform 1s;
+    transform-origin: left top;
+    transform-style: preserve-3D;
+    transition-duration: 2s;
+}
+
```

Comment:
can you create common css property for transition-duration, transform-style, transition, looks like it's common property for all box